peachjam/models/settings.py

```diff
@@ -72,6 +72,11 @@ class PeachJamSettings(SingletonModel):
         default=False,
         help_text=_("Allow signups via social accounts"),
     )
+    allow_signups = models.BooleanField(
+        verbose_name=_("allow signups"),
+        default=True,
+        help_text=_("Allow users to create accounts"),
+    )
```

Comment:
i wish we could disambiguate this and the other setting above. the two are doing similar things that could be rolled into one setting.

Reply:
ja, I tried to find a combined setting in django-allauth, but couldn't find one. I've pushed a change to actually enforce this (based on https://docs.allauth.org/en/dev/account/adapter.html#allauth.account.adapter.DefaultAccountAdapter.is_open_for_signup )